app/services/slack_notification_service.rb

```diff
@@ -16,9 +20,53 @@ def send_notification
 
   private
 
-  def notify_open_pull_request
+  def notify_pull_request
+    pull_request_url = extra_params[:pull_request][:html_url]
+    client = Slack::Web::Client.new
+    client.chat_postMessage(channel: CHANNEL, text: pull_request_url)
+  end
+  
+  def filter_unlabeled_action
+    label_name = extra_params[:github_webhook][:label][:name]
+    if label_name == ON_HOLD
```

Comment:
you should check all the variations of on hold ("on Hold", "On Hold", etc), so with `label_name. downcase == ON_HOLD.downcase` will be enough